src/routes/fest-route/fest-route.jsx

```diff
@@ -1,16 +1,18 @@
 import { useMutation, useQuery } from "@apollo/client";
 import { useMemo } from "react";
-import { useParams } from "react-router";
+import { useNavigate, useParams } from "react-router";
 import { Button, Header, Loading, MovieCardList } from "../../components";
 import { GET_FEST, GET_MOVIES } from "../../graphql/";
 import { UPDATE_FEST } from "../../graphql/mutations/fest";
-import { useModals } from "../../hooks";
+import { useCurrentUser, useModals } from "../../hooks";
 import magGlassDark from "../../images/mag-glass-black.svg";
 import { FestHeader, FestModal, FestSidebar } from "../fest-route";
 
 export const FestRoute = () => {
+  const { currentUser } = useCurrentUser();
```

Comment:
It looks like `currentUser` and `router` below aren't being used so can be removed, including their respective imports above.

Reply:
Ok, I've updated the branch to remove those variables!